lib/compress/zstd_compress_internal.h

```diff
@@ -29,6 +29,12 @@ extern "C" {
 #endif
 
 
+/* Dedicated dict search bucket log:
+ * ---------------------------------
+ * This determines the additional space we need for the hash table.
+ * We will have 2^DD_BLOG slots in our bucket. */
+#define DD_BLOG 2
```

Comment:
The source code comment is fine,
but the name itself is not terribly self-descriptive.
I would suggest something longer, like ` DDS_BUCKETLOG`.

Reply:
It should also start with `ZSTD_`, so there is no chance of collision with other macros.